src/makeCalculator.js

```diff
@@ -37,7 +37,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
```

Comment:
You change the result value in every methods. Will be better to do it only in operate method. In this method just return a + b